daml-assistant/get-daml.sh

```diff
@@ -65,6 +61,13 @@ readonly URL="https://github.com/digital-asset/daml/releases/download/v$VERSION/
 echo "$(tput setaf 3)Downloading latest DAML SDK. This may take a while.$(tput sgr 0)"
 curl -L $URL --output $TARBALL --progress-bar
 
+readonly DAML_HOME="$HOME/.daml"
+if [ -d $DAML_HOME ] ; then
+  echo "Removing existing installation."
```

Comment:
I wonder if we should ask for confirmation before doing that. But at least for now, the only reason not to reinstall is bandwidth as the config file doesn’t contain anything particularly interesting so it’s probably fine

Reply:
Having a no prompt ever solution is much nicer for scripting this into a CI, which is where I'd want to use it.

Reply:
And removing the existing installation forcefully is a good default here since it means the user can't get into a state where `curl get-daml.sh | sh` won't solve their problem.

Should I make an effort to preserve their config file nonetheless? Doing so *safely* probably means feeding the old config (if it exists) as an argument to `daml install`.

Reply:
I think my preference would be yes, but at the same time, that sets you up for weird failures where the problem you were seeing is the old config file being wacky. How about we don't for now, and add the complexity later if users want it?

Reply:
:+1: sounds good

Reply:
:+1: